# variables to hold HSV settings

int64[6] hsv_values 